File: dotnet_api/ConfigService.cs
using dotnet_api;

public interface IConfigService
{
    Task First(RequestInfoDataRequest data);
    Task Second(RequestInfoDataRequest data);
    Task Third(RequestInfoDataRequest data);
}

public class ConfigService : IConfigService
{
    private readonly IRepository _repository;

    public ConfigService(IRepository repository)
    {
        this._repository = repository;
    }

    public async Task First(RequestInfoDataRequest data)
    {
        var name_list = new List<string> { "first_first", "first_second" };
        await this._repository.ExecuteFunction(data, name_list);
    }

    public async Task Second(RequestInfoDataRequest data)
    {
        var name_list = new List<string> { "second_first", "second_second" };
        await this._repository.ExecuteFunction(data, name_list);
    }

    public async Task Third(RequestInfoDataRequest data)
    {
        var name_list = new List<string> { "third_first", "third_second" };
        await this._repository.ExecuteFunction(data, name_list);
    }
}

File: flask_api/config_service.py
from repository import Repository

def first_config_service(datas):
    name_list = ['first_first', 'first_second']

    return Repository().execute_action(datas, name_list)

def second_config_service(datas):
    name_list = ['second_first', 'second_second']

    return Repository().execute_action(datas, name_list)

def third_config_service(datas):
    name_list = ['third_first', 'third_second']

    return Repository().execute_action(datas, name_list)
